README.md

```diff
@@ -8,8 +8,27 @@
 
 ##### Build/Deploy
 
+Create a `config/prod.secret.exs` (see
+[example.prod.secret.exs](./config/example.prod.secret.exs))
+
+Then do:
+
 `MIX_ENV=prod mix do compile, release --env=prod --verbose`
 
+##### Build a Docker image
```

Comment:
i'm moving the colon into the bracket.  that will signify the tag is optional.
they will be building with the default :latest tag if they build it without the tag option

also, that documentation is for Docker.  i don't think it's necessary for us to include usage info for the Docker tools